lib/matplotlib/tests/test_bezier.py

```diff
@@ -0,0 +1,25 @@
+from matplotlib.tests.test_path import _test_curves
+
+import numpy as np
+import pytest
+import scipy.integrate
```

Comment:
you need cleaner erroring-out if scipy is not installed (see how we do it with pandas, or you can use a straight pytest.importorskip).

Reply:
Thanks for the pointers!